src/main/java/at/hannibal2/skyhanni/utils/StringUtils.kt

```diff
@@ -334,6 +334,11 @@ object StringUtils {
 
     fun generateRandomId() = UUID.randomUUID().toString()
 
+    fun generateRandomString(length: Int): String {
+        val chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789"
```

Comment:
store the char array otuside the function so it does not get created on each function call